test/test.js

```diff
@@ -1,5 +1,9 @@
 var TodoList = require('../src/javascripts/todo-list.js')
 
+var mocha = require('mocha')
+
+var describe = mocha.describe
```

Comment:
@Prerna1993 As far as I am aware, mocha does not contain the describe method. Could you link me to the docuentation where you found describe to be a part of mocha?